.azure-pipelines/release-cli.yaml

```diff
@@ -55,659 +158,604 @@ parameters:
+                    displayName: Compress signed files (Windows)
+                    condition: and(succeeded(), eq(variables['SHOULD_SIGN'], 'True'))
+
+                - ${{ if startsWith(pool.rid, 'osx') }}:
+                  - task: PowerShell@2
+                    inputs:
+                      pwsh: true
+                      targetType: inline
+                      script: |
+                        . $(powershellScriptsDir)/BuildTools.ps1
+                        $zipPath = Join-Path -Path '$(SIGN_PATH)' -ChildPath '$(ZIP_NAME)'
+                        $fileName = Get-FileName -FileNameTemplate '$(fileNameTemplate)' -BranchOrTagName '$(branchOrTagName)' -RuntimeIdentifier '$(rid)'
+                        Update-SignedArchive -InputFile $zipPath -ReportDir '$(WORKING_DIR)' -OutputDir '$(WORKING_DIR)' -OutputFileName "$fileName" -ExeNames mgc -Cleanup
+                      verbosePreference: '$(OUTPUT_PREFERENCE)'
+                      debugPreference: '$(OUTPUT_PREFERENCE)'
+                      informationPreference: '$(OUTPUT_PREFERENCE)'
+                    displayName: Update signed files (MacOS)
+                    condition: and(succeeded(), eq(variables['SHOULD_SIGN'], 'True'))
+
+              # https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/1es-pipeline-templates/features/outputs
+              templateContext:
+                outputs:
+                - output: pipelineArtifact
+                  condition: and(succeeded(), eq(variables['SHOULD_SIGN'], 'True'))
+                  displayName: Upload signed output
+                  artifact: sign-output-$(rid)
+                  path: $(WORKING_DIR)
+
+      - stage: upload
+        dependsOn: [sign]
+        condition: and(succeeded(), ne('${{ parameters.simulate }}', 'true'))
+        jobs:
+          - job: uploadGitHub
+            displayName: Upload binaries (GitHub)
+            # Only upload release if we're building a tag.
+            condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/tags/v'))
+            steps:
+              - checkout: none
+              - task: DownloadPipelineArtifact@2
+                inputs:
+                  path: $(artifactsDownloadLocation)
+              - task: GithubRelease@1
+                displayName: 'Upload Artifacts to GitHub Release'
+                inputs:
+                  gitHubConnection: $(repositoryConnection)
+                  repositoryName: '$(Build.Repository.Name)'
+                  action: edit
+                  tag: $(branchOrTagName)
+                  addChangeLog: false
+                  assetUploadMode: replace
+                  assets: |
+                    $(artifactsDownloadLocation)/build-output-linux-*/*.tar*
+                    $(artifactsDownloadLocation)/sign-output-*/*.tar*
+                    $(artifactsDownloadLocation)/sign-output-*/*.zip
+
+          ## Uncomment when NuGet deployment is re-enabled
```

Comment:
should this be uncommented before merge?

Reply:
NuGet deployment was disabled until we solve #379.

Reply:
right, sorry, I forgot about that. Ignore my tired self.